<launch>
    <node name="image_publisher_node" pkg="drive_ros_image_from_files" type="image_publisher_node" respawn="false" output="screen">
        <param name="image_folder" type="string" value="/path/to/images" />
        <param name="publish_rate" type="double" value="10" />
        <remap from="img_out" to="camera_image"/>
    </node>
</launch>
